test/views/coronavirus_local_restrictions_test.rb

```diff
@@ -0,0 +1,90 @@
+require "test_helper"
+require "gds_api/test_helpers/mapit"
+
+class CoronavirusLocalRestrictionTest < ActionView::TestCase
```

Comment:
It'll be good to think about how the name (and filename) of a view test maps into the wider test structure.

With [RSpec's approach to view specs](https://relishapp.com/rspec/rspec-rails/v/2-3/docs/view-specs/view-spec) the tests would be in a file of `spec/views/coronavirus_local_restrictions/results.html.erb_spec.rb` so there is an individual test per view.

I think here we should decide if an individual view has it's own test or whether each view inside a directory has a describe block (as per controllers)

e.g.

```
# tests/views/coronavirus_local_restrictions/results_html_test.rb
class CoronavirusLocalRestrictions::ResultsHtmlTest < ActionView::TestCase
  test "renders postcode match tier 4" 
end
```

or 

```
# tests/views/coronavirus_local_restrictions_test.rb
class CoronavirusLocalRestrictionsTest < ActionView::TestCase
  describe "results.html.erb" do
    test "renders postcode match tier 4" 
  end
end
```

I think the first approach of test per view is probably better as it is less likely to have a namespace collision, can scale easier and produces more short tests compared to one big test. It'd also make it easier to port to RSpec.

Reply:
Thanks! I have updated.